index.html

```diff
@@ -110,21 +110,23 @@ <h1></h1>
         <div id="pickerDiv">  
         </div></div>
 
+        <div id="wordInputHeader">
```

Comment:
too many closing divs

Reply:
I'll take a look at it.

Reply:
To be fair, I didn't even write that one x_x